proof/invariant-abstract/X64/ArchInvariants_AI.thy

```diff
@@ -3371,6 +3371,10 @@ proof -
   thus ?thesis by (rule rtrancl_mono)
 qed
 
+(* unused on this architecture *)
+declare is_irq_control_descendant_def[simp]
+declare should_be_arch_parent_of_def[simp]
```

Comment:
Should we do this in RISCV64 as well? It might make it easier to keep the proofs consistent